0.wave1_data/4.analyze_wave1/nbconverted/gasdermin_intensity.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Determine the distribution of Gasdermin intensity
+# 
+# - Across treatments per dilation experiment
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+import plotnine as gg
+
+
+# In[2]:
+
+
+# Load single cell profiles
+feature_dir = pathlib.Path("..", "3.process_features", "data")
+
+# This determines the file
+file_prefix = "interstellar_wave1_dilate"
+file_suffix = "_sc.csv.gz"
+dilation_factors = [25, 50, 100]
+
+# Determine output figure directory
+figure_dir = "figures"
+
+
+# In[3]:
+
+
+# Create three figures per dilation experiment
+for dilation_factor in dilation_factors:
+    cp_file = pathlib.Path(feature_dir, f"{file_prefix}{dilation_factor}{file_suffix}")
+
+    # Load data
+    cp_df = pd.read_csv(cp_file, low_memory=False)
+
+    # Ensure dose is a string and recode to high/low
+    cp_df.Metadata_dose = cp_df.Metadata_dose.astype(str)
+
+    dose_recode = {
+        "0": "low",
+        "0.1µM": "low",
+        "1µM": "low",
+        "1µg/ml + 1µM": "low",
+        "2.5µM": "high",
+        "10µM": "high",
+        "1µg/ml + 10µM": "high"
+    }
+    cp_df = cp_df.assign(Metadata_dose_recode=cp_df.Metadata_dose.replace(dose_recode))
+
+    # Recode number of neighbors
+    median_neighbors = (
+        cp_df.TranslocatedNuclei_Neighbors_NumberOfNeighbors_Expanded.median()
+    )
+
+    cp_df = cp_df.assign(Metadata_neighbor_recode="high")
+    (
+        cp_df
+        .loc[
+            cp_df.TranslocatedNuclei_Neighbors_NumberOfNeighbors_Expanded < median_neighbors,
+            "Metadata_neighbor_recode"
+        ]
+    ) = "low"
+
+    # Create variable for translocation ratio
+    cp_df = (
+        cp_df
+        .assign(
+            translocation_ratio = (
+                cp_df.TranslocatedNuclei_Intensity_IntegratedIntensity_CorrGasderminD / 
+                cp_df.DilatedNuclei_Intensity_IntegratedIntensity_CorrGasderminD
```

Comment:
This is probably a very easy question, but how do these features divided represent a translocation ratio?

Reply:
Based on the figure (https://github.com/WayScience/Interstellar_Project/blob/main/0.wave1_data/2.cellprofiler_analysis/figures/dilation_ex.png) the translocated nuclei object is our "cytoplasm" (nuclei excluded) and dilated nuclei object is our "cells" (nuclei included).

My thinking was that the ratio of "cytoplasm" gasdermin over total gasdermin represents the proportion of translocated gasdermin per cell.

I think this makes sense, but please correct me if I'm getting mixed up!

Reply:
I'm going to merge for now, but I can make changes later if needed.